scripts/end_to_end_test/smart_contract_tests/transfer.py

```diff
@@ -71,6 +71,9 @@ def setup(api):
     api.sync(api.contracts.create(entity1, contract1, 2000))
     api.sync(api.contracts.create(entity1, contract2, 2000))
 
+    api.sync(api.tokens.transfer(entity1, contract1.address, 10000, 500))
+    api.sync(api.tokens.transfer(entity1, contract2.address, 10000, 500))
```

Comment:
Try to remove this as discussed. If that causes this test to fail, then something fishy is going on.